sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -129,37 +129,37 @@ The Text Analytics client library provides a [TextAnalyticsClient][text_analytic
 [TextAnalyticsAsyncClient][text_analytics_async_client] to do analysis on batches of documents. It provides both synchronous and
 asynchronous operations to access a specific use of Text Analytics, such as language detection or key phrase extraction.
 
-### Text input
-A **text input**, sometimes called a **document**, is a single unit of input to be analyzed by the predictive models
-in the Text Analytics service. Operations on Text Analytics client may take a single text input or a collection
-of inputs to be analyzed as a batch. 
+### input
```

Comment:
Should've pointed it out earlier, but I think it would be better if it was like:

### Document
A **document** is a text input to be analyzed by the predictive models

Reply:
I think all of us are using title `Input`, (p.s., I forgot to capitalize the first letter):

https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/textanalytics/Azure.AI.TextAnalytics#input
https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/textanalytics/azure-ai-textanalytics#input